src/main/ctl/git.ts

```diff
@@ -11,29 +11,40 @@ class Git {
   user: string;
   project: string;
   args: string[];
-
+  isGithub: boolean = true;
```

Comment:
Consider using `gitProvider` instead because aside GitHub and GitLab, there are other git providers like Bitbucket, AWS CodeCommit, Coding, etc.